source/simulator/core.cc

```diff
@@ -2023,6 +2025,8 @@ namespace aspect
     CitationInfo::print_info_block (pcout);
 
     stokes_matrix_free.reset();
+
+    pcout << "-- Total wallclock time elapsed including restarts:" << round(walltime.wall_time()+total_walltime_since_last_snapshot) << "s" << std::endl;
```

Comment:
Is there a reason you put this not directly after the `print_summary` statement in line 2023? In the current form you will have the citation information between the last print of the computing_timer and this new information. Does this form look better?

Reply:
sure, it's better to move it to line 2023